CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## Changelog
 
+## v6.1.1
+
+- fix `bicycle-share-11` icon path to get optimized correctly downstream
```

Comment:
I know this is nitpicky, but I don't think you need to include this downstream context. _fix `bicycle-share-11` icon path_ seems good to me.